pom.xml

```diff
@@ -510,7 +514,7 @@
           <sortDependencyExclusions>groupId,artifactId</sortDependencyExclusions>
           <sortExecutions>true</sortExecutions>
           <sortModules>true</sortModules>
-          <sortPlugins>groupId,artifactId</sortPlugins>
+          <sortPlugins>groupId</sortPlugins>
```

Comment:
When two plugins have an execution in the same phase, the plugins run in order they appear in pom.xml. This was problematic for the antrun plugin and had to be changed